download.dd

```diff
@@ -204,8 +204,8 @@ Macros:
 
     DMDV2=$(LATEST)
 
-    _=BETA=$(COMMENT $0)
-    BETA=$0
+    BETA=$(COMMENT $0)
+    _=BETA=$0
```

Comment:
We can automate this too if you want to.

Reply:
It's already automated.